t/shell/mpir.c

```diff
@@ -15,6 +15,7 @@
 #endif
 
 #include <flux/core.h>
+#include <flux/optparse.h>
```

Comment:
In "testsuite: switch mpir test program to optparse " commit message, s/--leade-rank/leader-rank/